lxd/cgroup/abstraction.go

```diff
@@ -827,6 +827,46 @@ func (cg *CGroup) GetMemoryStats() (map[string]uint64, error) {
 	return out, nil
 }
 
+// GetOOMKills returns the number of oom kills.
+func (cg *CGroup) GetOOMKills() (int64, error) {
+	var (
+		err   error
+		stats string
+	)
+
+	version := cgControllers["memory"]
+
+	switch version {
+	case Unavailable:
+		return -1, ErrControllerMissing
+	case V1:
+		stats, err = cg.rw.Get(version, "memory", "memory.oom_control")
+	case V2:
+		stats, err = cg.rw.Get(version, "memory", "memory.events")
+	}
+
+	if err != nil {
+		return -1, err
+	}
+
+	for _, stat := range strings.Split(stats, "\n") {
+		field := strings.Split(stat, " ")
+		// skip incorrect lines
+		if len(field) != 2 {
+			continue
+		}
+
+		switch field[0] {
+		case "oom_kill":
+			out, _ := strconv.ParseInt(field[1], 10, 64)
+
+			return out, nil
+		}
+	}
+
+	return -1, nil
```

Comment:
What does -1 with no error here indicate?

Reply:
line with "oom_kill" not found.

I will add an error here.

Reply:
done